### PR TITLE
Add PerfContext counters for CompressedSecondaryCache

### PR DESCRIPTION
**Pull request by gitbw95**
Summary:
Add PerfContext counters for CompressedSecondaryCache.

Test Plan:
Unit Tests.

### CODE REVIEW

File: cache/lru_cache.cc [hunk 1]
```diff
@@ -458,6 +458,8 @@ void LRUCacheShard::Promote(LRUHandle* e) {
         e->Unref();
         e->Free();
         e = nullptr;
+      } else {
+        PERF_COUNTER_ADD(block_cache_standalone_handle_count, 1);
```
Comment (anand1976) [inline cache/lru_cache.cc:462]: Should we also have a counter for number of handles converted form standalone to proper entries?

File: include/rocksdb/perf_context.h [hunk 2]
```diff
@@ -69,13 +69,23 @@ struct PerfContext {
...
+  uint64_t block_cache_standalone_handle_count;
   uint64_t index_block_read_count;        // total number of index block reads
   uint64_t block_cache_filter_hit_count;  // total number of filter block hits
   uint64_t filter_block_read_count;       // total number of filter block reads
   uint64_t compression_dict_block_read_count;  // total number of compression
                                                // dictionary block reads
 
   uint64_t secondary_cache_hit_count;  // total number of secondary cache hits
+  // total number of real handles inserted into secondary cache
+  uint64_t secondary_cache_insert_real_count;
```
Comment (anand1976) [inline include/rocksdb/perf_context.h:82]: Can we call these ```compressed_cache_*```?

**Review by anand1976 — COMMENTED** [review]
How will these counters be exported to ods and/or scuba?

File: cache/lru_cache.cc (hunk 1 above)
Comment (gitbw95) [reply cache/lru_cache.cc:462]: There are two type of real handles from secondary cache:
1. standalone handles, with a dummy handle inserted into primary cache.
2. real handles that are inserted into primary cache.

secondary_cache_hit_count = # of standalone handles + # of real handles.
I can add "# of real handles".

**Review by anand1976 — APPROVED** [review]

**Pull request closed without merging**